### ISSUE TITLE
Weird behavior of j/k command for moving to the next/previous cell 

### ISSUE THREAD

**Issue by liganega** [microsoft/vscode-jupyter#12738]
### Applies To

- [X] Notebooks (.ipynb files)
- [ ] Interactive Window and\/or Cell Scripts (.py files with \#%% markers)

### What happened?

This is a report of a weird bevior of the (vim-style) j/k command.
First, it's only related to markdown cells.

Description of the weird behavior after editing a **markdown cell**:

- When Control+Enter keybinding is applied, **the j/k command won't work at all.**
- When other keybindings such as  ESC key or Shift+Enter are applied, then it works as expected.

This behavior has been confirmed on several (Windows 11) PCs, even with reinstalled VS Code + Python extensions.
With code cells, on the other hand, there is no problem.

### VS Code Version

Version: 1.75.0 (user setup) Commit: e2816fe719a4026ffa1ee0189dc89bdfdbafb164 Date: 2023-02-01T15:23:45.584Z Electron: 19.1.9 Chromium: 102.0.5005.194 Node.js: 16.14.2 V8: 10.2.154.23-electron.0 OS: Windows_NT x64 10.0.22621 Sandboxed: No

### Jupyter Extension Version

v2023.1.2000312134

### Coding Language and Runtime Version

Python 3.9.12

### Anaconda Version (if applicable)

Anaconda3-2022.10-Windows-x86_64.exe

### Running Jupyter locally or remotely?

None

**Comment by amunger** [comment]
> When Control+Enter keybinding is applied, the j/k command won't work at all.

Can you clarify the repro steps here? If I press `ctrl+enter` to render a markdown cell, j/k still navigates between cells for me as expected.

**Comment by liganega** [comment]
I don't have any repro steps, but I tested again using another **Windows 10 and 11 PCs (Korean edition)**. (not mine)
Here is what I did:

- VS Code and Anaconda reinstalled. (all the configs were removed first)
- Open VS Code and create a new ipynb file.
- Install all extensions what VS Code recommends: Jupyter, Jupyter Cell Tags, Jupyter Keymap, Jupyter Notebook Renderers, Jupyter Slide Show, Pylance, Python

That's all. But pressing `ctrl+enter` to render a markdown cell won't let me navigate down/up using j/k key strokes.

**Comment by amunger** [comment]
ah, I got it to repro now. I was trying with the vim extension installed, I forgot that the j/k keyboard commands are available just with the jupyter extension

**Comment by amunger** [comment]
actually, all notebook cell commands are disabled (a, b, dd) as noted in the linked issue.

**Comment by liganega** [comment]
Is this fixed, or will it be?

I hope so because this is really so annoying that I don't want to use VS Code with jupyter notebook anymore, almost.

**Comment by rebornix** [comment]
I can reproduce this issue, working on a fixed now.

**Comment by jrounds** [comment]
I had this issue and the key debug logger said arrow keys were hitting "scrollLeftHover", "scrollRightHover", "scrollUpHover", and "scrollDownHover".  I remapped those events not to be arrow keys and arrow keys returned to normal function in Jupyter notebook.  I assume this is the same issue?

**Comment by RishabhSk7** [comment]
@jrounds yeah getting same logs in the logger. Might possibly have been due to a sublime key binding extension if I had to guess. 
If anyone is not sure how to remap key events, go to file->preferences->Keyboard Shortcuts.
then search for "scrollLeftHover", "scrollRightHover", "scrollUpHover", and "scrollDownHover" and delete the respective entries.

**Comment by rebornix** [comment]
This issue will be fixed and released in tomorrow's VS Code Insiders. Thank you for the input.

**Comment by liganega** [comment]
Thank you for your effort!
